Configurations/unix-Makefile.tmpl

```diff
@@ -961,7 +961,7 @@ reconfigure reconf:
           my $ord_name = $args{generator}->[1] || $args{product};
           return <<"EOF";
 $target: $args{generator}->[0] $deps \$(SRCDIR)/util/mkdef.pl
-	\$(PERL) \$(SRCDIR)/util/mkdef.pl$ord_ver --ordinals $args{generator}->[0]  --name $ord_name --OS $mkdef_os > $target
+	\$(PERL) "\$(SRCDIR)/util/mkdef.pl"$ord_ver --ordinals $args{generator}->[0]  --name $ord_name --OS $mkdef_os > $target
```

Comment:
is there a reason why you placed the quote _before_ $ord_ver ?

Reply:
`$ord_ver` is not supposed to be part of the filename, as far as I could read from the code. Its value is either ` --version $(VERSION)` or empty.

Reply:
Ugh!